backstory/pom.xml

```diff
@@ -35,7 +35,34 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-all</artifactId>
+      <version>1.9.5</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.github.origma</groupId>
+      <artifactId>Google-PerspectiveAPI-Java-Client</artifactId>
+      <version>0.0.5</version>
+    </dependency>
   </dependencies>
+  
+  <repositories>
+    <repository>
+      <id>jitpack.io</id>
+      <url>https://jitpack.io</url>
```

Comment:
Out of curiosity, what is this used for?

Reply:
This should have been removed. It was used for CSV processing, and it accidentally got copied over from the last project because I forgot to check the repositories, as this was the only one. I will go remove it. Thank you.

Reply:
This is used for the Java client for Perspective API. The [GitHub repository](https://github.com/origma/Google-PerspectiveAPI-Java-Client) recommends having this as well as the dependency for the repo itself.

Reply:
Done